Application/EdFi.Admin.DataAccess/EdFi.Admin.DataAccess.csproj

```diff
@@ -21,5 +22,6 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\EdFi.Common\EdFi.Common.csproj" />
+    <None Include="../../LICENSE.txt" Pack="true" PackagePath="LICENSE.txt" />
```

Comment:
this should be in another item group. lets keep project references together in their own group

Reply:
Fixed it . Thanks